src/plugins/base-notifiers.js

```diff
@@ -232,11 +250,13 @@ function plugin(loggin) {
         }
     }
 
-    Notifier.register('Console', ConsoleNotifier);
-    Notifier.register('File', FileNotifier);
-    Notifier.register('Http', HttpNotifier);
-    Notifier.register('Remote', RemoteNotifier);
-    Notifier.register('Memory', MemoryNotifier);
-};
+    notifierRegistry
+        .register('Console', ConsoleNotifier)
+        .register('Default', ConsoleNotifier)
+        .register('File', FileNotifier)
+        .register('Http', HttpNotifier)
+        .register('Remote', RemoteNotifier)
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/loggin-js/loggin-js/pullRequest?prid=7629800)
```suggestion
        .register("Remote", RemoteNotifier)
```